en/docs/guides/login/query-saml-assertions.md

```diff
@@ -2,3 +2,63 @@
 
 This page guides you through querying dynamic or existing [SAML2 assertions](insertlink) using standard request messages via WSO2 Identity Server. 
 
+----
+
+This guide assumes you have your own application. If you wish to try out this flow with a sample application click the button below. 
+
+<a class="samplebtn_a" href="../../../quick-starts/query-saml-assertions-sample" target="_blank" rel="nofollow noopener">Try it with the sample</a>
```

Comment:
If you use target="_blank" it will open a new window right. We need to avoid this and have proper links in the bottom of the page.